doc/start/quick-cephfs.rst

```diff
@@ -3,117 +3,116 @@
    clean``  state. Also, ensure that you have at least one :term:`Ceph Metadata
-   Server` running. :: 
+   Server` running. ::
 
 	ceph -s [-m {monitor-ip-address}] [-k {path/to/ceph.client.admin.keyring}]
 
 
-Create a Filesystem
-===================
+Create a File System
+====================
 
 You have already created an MDS (`Storage Cluster Quick Start`_) but it will not
-become active until you create some pools and a file system.  See :doc:`/cephfs/createfs`.
+become active until you create some pools and a file system.  See
+:doc:`/cephfs/createfs`.::
 
-::
-
-    ceph osd pool create cephfs_data <pg_num>
-    ceph osd pool create cephfs_metadata <pg_num>
+    ceph osd pool create cephfs_data 32
+    ceph osd pool create cephfs_metadata 32
     ceph fs new <fs_name> cephfs_metadata cephfs_data
 
+Quick word about Pools and PGs
+------------------------------
 
-Create a Secret File
-====================
-
-The Ceph Storage Cluster runs with authentication turned on by default. 
-You should have a file containing the secret key (i.e., not the keyring 
-itself). To obtain the secret key for a particular user, perform the 
-following procedure: 
+Since the default replication number/size is 3, you'd need 3 OSDs to get
+``active+clean`` for all PGs. Alternatively, you may change the replication
+number for the pool to match the number of OSDs::
 
-#. Identify a key for a user within a keyring file. For example:: 
+    ceph osd pool set cephfs_data size {number-of-osds}
+    ceph osd pool set cephfs_meta size {number-of-osds}
 
-	cat ceph.client.admin.keyring
+And, in case you have deployed all of the OSDs on the same node, you would need
+to create a new CRUSH rule to replicate data across OSDs and set the rule on the
+CephFS pools, since the default CRUSH rule is to replicate data across
+different nodes::
 
-#. Copy the key of the user who will be using the mounted CephFS file system.
-   It should look something like this:: 
-	
-	[client.admin]
-	   key = AQCj2YpRiAe6CxAA7/ETt7Hcl9IyxyYciVs47w==
+    ceph osd crush rule create-replicated rule_foo default osd
+    ceph osd pool set cephfs_data crush_rule rule_foo
+    ceph osd pool set cephfs_meta crush_rule rule_foo
 
-#. Open a text editor. 
+Usually, setting ``pg_num`` to 32 gives a perfectly healthy cluster. To pick
+appropriate value for ``pg_num``, refer `Placement Group`_.
 
-#. Paste the key into an empty file. It should look something like this::
+You can also use pg_autoscaler plugin instead. Introduced by Nautilus release,
+it can automatically increase/decrease value of ``pg_num``; refer the
+`Placement Group`_ to find out more about it.
 
-	AQCj2YpRiAe6CxAA7/ETt7Hcl9IyxyYciVs47w==
-
-#. Save the file with the user ``name`` as an attribute 
-   (e.g., ``admin.secret``).
-
-#. Ensure the file permissions are appropriate for the user, but not
-   visible to other users. 
-
-
-Kernel Driver
-=============
+Mounting the File System
+========================
```

Comment:
Maybe include how to specify an alternate file system.

Reply:
Added a sample commands for both kernel and FUSE that shows how to do that.